### ISSUE TITLE
Formula highlight bug under editing mode

### ISSUE THREAD

**Issue by nedchu** [tamlok/vnote#291]
In VNote version 1.19. Something goes wrong in highlighting of editing mode. But the formula can be successfully rendered in reading mode.
![image](https://user-images.githubusercontent.com/11944144/42316732-1a261338-807d-11e8-95ce-758e014c3eb3.png)
![image](https://user-images.githubusercontent.com/11944144/42316909-8dced20c-807d-11e8-9020-aad80de3368b.png)

This is raw text:
``` latex
- 推广到二维可以得到平均滤波$G(i,j)=\frac{1}{(2k+1)^2}\sum\limits_{u=-k}^k{\sum\limits_{v=-k}^k{F[i+u,j+v]}}$
- 推广到
```

**Comment by tamlok** [comment]
Please use `\` to escape `_`. Edit mode is stricter than read mode. Efforts are needed to make them behave identically.

**Comment by tamlok** [comment]
Should be fixed in 1.20 now. Try it out.
